charts/spire/charts/spire-server/values.yaml

```diff
@@ -201,6 +221,24 @@ telemetry:
       namespace: ""
       labels: {}
 
+ingress:
+  enabled: false
+  className: ""
```

Comment:
Maybe `spire-server`?

Reply:
For enabled?

Its hard to read in the patch, but its already spire-server.ingress.enabled.

For this one, there is a slight chance it could be confused with the other ingress(s) in the chart. so 
spire-server.server.ingress.enabled could disambiguate. but may also be more confusing then its worth?

Reply:
Agree to leave it as is, we are already in the context of the spire-server chart.